pkg/crc/api/api_http.go

```diff
@@ -49,15 +50,22 @@ func setPullSecret() func(c *context) error {
 		if err := cluster.StoreInKeyring(string(c.requestBody)); err != nil {
 			return err
 		}
-		return c.String(http.StatusCreated, "")
+		return c.JSON(http.StatusCreated, client.Result{
+			Success: true,
+		})
 	}
 }
 
 func getPullSecret(config crcConfig.Storage) func(c *context) error {
 	return func(c *context) error {
 		if _, err := cluster.NewNonInteractivePullSecretLoader(config, "").Value(); err == nil {
-			return c.String(http.StatusOK, "")
+			return c.JSON(http.StatusOK, client.Result{
+				Success: true,
```

Comment:
Maybe we can send a useful information about the pull secret here? Like the username of the user, the size of the pull secret, the pull secret is valid: true|false. Success seems redundant.

It is just a proposal ^^

Reply:
Would not be a good idea. This can be regarded of leaking PII.

I would either, if an actual JSON object is requested, to just return `{ }`